.template/addons/github/.github/workflows/test.yml.tt

```diff
@@ -0,0 +1,76 @@
+name: Test
+
+on: push
+
+env:
+  DOCKER_REGISTRY_HOST: ${{ secrets.DOCKER_REGISTRY_HOST }}
+  DOCKER_REGISTRY_USERNAME: ${{ github.repository_owner }}
+  DOCKER_REGISTRY_TOKEN: ${{ secrets.DOCKER_REGISTRY_TOKEN }}
+  COMPOSE_FILE: docker-compose.test.yml
+  DOCKER_IMAGE: ${{ github.repository }}
+
+jobs:
+  build:
+    name: Build docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cancel previous runs
+        uses: styfle/cancel-workflow-action@0.5.0
+        with:
+          access_token: ${{ github.token }}
+
+      - uses: actions/checkout@v2.3.2
+
+      - uses: nimblehq/branch-tag-action@v1.1
+
+      - name: Docker login
+        run: echo "$DOCKER_REGISTRY_TOKEN" | docker login $DOCKER_REGISTRY_HOST --username=$DOCKER_REGISTRY_USERNAME --password-stdin
+
+      - name: Docker pull
+        if: ${{ env.BRANCH_TAG != 'latest' && env.BRANCH_TAG != 'development' }}
```

Comment:
This syntax isn't wrong but can simplify it as
```yml
if: env.BRANCH_TAG != 'latest' && env.BRANCH_TAG != 'development'
```

according to the [official documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#about-contexts-and-expressions):
> When you use expressions in an `if` conditional, you may omit the expression syntax (${{ }}) because GitHub automatically evaluates the if conditional as an expression. For more information about if conditionals, see "Workflow syntax for GitHub Actions."

Reply:
Yes, we can simplify it, but IMO, keeping it with `${{ }}` syntax will help us notice that we're using the expression of GitHub action, and make them consistent with other places using the expression.
 What do you think?